File: openapi/code_samples/go/api_v1_organizations_{organization_id}_clients_{client_id}/patch.go
response, err := scalekitClient.M2M().UpdateOrganizationClient(
  ctx,
  "org_123",
  "skc_xxxxx",
  scalekit.UpdateOrganizationClientOptions{
    Description: "Service account for GitHub Actions to deploy applications to production_eu",
  },
)
if err != nil {
  // handle error
  return
}
_ = response

File: openapi/code_samples/java/api_v1_organizations_{organization_id}_clients_{client_id}/patch.java
import com.scalekit.grpc.scalekit.v1.clients.OrganizationClient;

OrganizationClient updates = OrganizationClient.newBuilder()
    .setDescription("Service account for GitHub Actions to deploy applications to production_eu")
    .build();

UpdateOrganizationClientResponse response = scalekitClient.m2m()
    .updateOrganizationClient("org_123", "skc_xxxxx", updates);
